isis/src/base/objs/ProcessExportPds4/ProcessExportPds4.cpp

```diff
@@ -52,6 +52,8 @@ namespace Isis {
 
     m_imageType = StandardImage;
 
+//    ProcessExport::setFormat(ExportFormat::BSQ); 
```

Comment:
Did this need to stay in here...?

Reply:
That should probably be removed.

Reply:
I thought about this, and it may be useful to keep in. I'm going to open up a ticket about a bug Paul Geissler found that was beyond the scope of this sprit (our exported labels look good*, but the actual img file cannot be opened using the pds4_viewer. This is also true for our existing, unmodified trurhdata, so it is not something we introduced during this sprint) and has to do with that commented-out line.

Reply:
Sounds good. Did you want to add comments explaining why it's there at all...?

Reply:
Yes, good call. Or, on second thought, I'll take this comment out and just put the info in a ticket.